doc/api/addons.md

```diff
@@ -367,6 +367,10 @@ set of APIs that are used by the native code. Instead of using the V8
 or [Native Abstractions for Node.js][] APIs, the functions available
 in the N-API are used.
 
+Creating and maintaining an add-on that benefits from the ABI stability
```

Comment:
`add-on` -> `addon` for consistency with the rest of the doc.

Reply:
Actually, I'll just go ahead and make that change myself since I'm right here...